### PR TITLE
adding site points shapefile to release

### PR DESCRIPTION
**Pull request by jread-usgs**
This is minor @aappling-usgs. Sets up https://www.sciencebase.gov/catalog/item/57ae2576e4b0fc09faad6e5c with both feature types

### CODE REVIEW

**Comment by jread-usgs** [comment]
Added two metadata file releases to this too - `basic`, and `dvqcoefs`

File: remake/4_data_release.yml [hunk 1]
```diff
@@ -22,27 +22,65 @@ targets:
...
 
   catchment_metadata:
     command: extract_feature(site_catchment_shapefile)
   
+  points_metadata:
+    command: extract_feature(site_points_shapefile)  
+  
   catchment_attr:
-    command: as.attr_list('../4_data_release/out/1_catchment_attr.tsv')
+    command: as.attr_list('../4_data_release/out/1_catchment_attr.csv')
```
Comment (aappling-usgs) [inline remake/4_data_release.yml]: csv is fine with me, but i thought you were a die-hard tsv fan...what changed?

File: remake/4_data_release.yml [hunk 2]
```diff
@@ -22,27 +22,65 @@ targets:
...
-  stream_metab_powell_release:
-    command: create_release_parent(target_name)
-
-  pc1_spatial:
-    command: create_release_child(stream_metab_powell_release, target_name)
-
+  
+  # -- meta --
+  ../4_data_release/out/1_meta_basic_attr.csv:
+    command: attribute_skeleton(meta_basic, target_name)
```
Comment (aappling-usgs) [inline remake/4_data_release.yml]: i'm a little wary of this skeleton+[re]make pattern. if we add a site to meta_basic, won't this wipe away our manual entries in the attr.csv? i know file versioning has our back, but i wish meddle could protect us from ourselves, too. not urgent but something to chew on.

File: 4_data_release/out/1_meta_basic_attr.csv [hunk 3]
```diff
@@ -0,0 +1,14 @@
+attr-label,attr-def,attr-defs,data-min,data-max,data-units
+site_name,the {site_database}_{site_num} for the site,National Water Information System,NA,NA,NA 
+site_database,source of dataset for this site,see National Water Information System,NA,NA,NA
```
Comment (aappling-usgs) [inline 4_data_release/out/1_meta_basic_attr.csv]: cool that NAs get plugged in automatically for strings

**Pull request merged**